lib/auth/auth.go

```diff
@@ -1077,10 +1077,6 @@ func (a *Server) ExtendWebSession(req WebSessionReq, identity tlsca.Identity) (s
 		return nil, trace.Wrap(err)
 	}
 
-	sess, err = services.ExtendWebSession(sess)
-	if err != nil {
-		return nil, trace.Wrap(err)
-	}
```

Comment:
Did you mean to delete this?

Reply:
yeah, see https://github.com/gravitational/teleport/pull/6685/files#diff-6d9bede626eef054444c2f0e375e87693700f2cd99dffa70e9681eebd2f697e8, that function was a no-op